### PR TITLE
updated caption jayeshsapariyanice

### PR DESCRIPTION
**Pull request by jayeshsapariyanice**
This PR updated the caption.
This PR fixes the issue.

## Done
-created branch
-made commit & Push

### CODE REVIEW

**Review by narsharma — APPROVED** [review]

File: _slides/09-jayeshsapariyanice.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 layout: slide
 title: "Welcome to our slide deck!"
```
Comment (ncsokas) [inline _slides/09-jayeshsapariyanice.md:3]: I like this title!

File: _slides/09-jayeshsapariyanice.md [hunk 2]
```diff
@@ -2,8 +2,7 @@
...
 ---
-
-Caption here
+Hello World!
```
Comment (ncsokas) [inline _slides/09-jayeshsapariyanice.md:5]:
```suggestion
Hello World! Nice to see you today!
```

**Review by ncsokas — APPROVED** [review]

File: _slides/09-jayeshsapariyanice.md (hunk 2 above)
Comment (AkshayJagtapNice) [reply _slides/09-jayeshsapariyanice.md:5]: Looks good

**Review by AkshayJagtapNice — APPROVED** [review]
PR Looks good

**Review by Vadim-Grinevich — APPROVED** [review]
kkkkk

**Review by AkshayJagtapNice — APPROVED** [review]

**Pull request merged**